src/pages/NewChatConfirmPage.tsx

```diff
@@ -0,0 +1,156 @@
+import React, {useMemo} from 'react';
+import {View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import type {OnyxEntry} from 'react-native-onyx';
+import Avatar from '@components/Avatar';
+import Badge from '@components/Badge';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import MenuItemWithTopDescription from '@components/MenuItemWithTopDescription';
+import ScreenWrapper from '@components/ScreenWrapper';
+import SelectionList from '@components/SelectionList';
+import TableListItem from '@components/SelectionList/TableListItem';
+import type {ListItem} from '@components/SelectionList/types';
+import useCurrentUserPersonalDetails from '@hooks/useCurrentUserPersonalDetails';
+import useLocalize from '@hooks/useLocalize';
+import useStyleUtils from '@hooks/useStyleUtils';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as GroupChatUtils from '@libs/GroupChatUtils';
+import Navigation from '@libs/Navigation/Navigation';
+import * as OptionsListUtils from '@libs/OptionsListUtils';
+import * as ReportUtils from '@libs/ReportUtils';
+import * as Report from '@userActions/Report';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+import type * as OnyxTypes from '@src/types/onyx';
+import type {Participant} from '@src/types/onyx/IOU';
+
+type NewChatConfirmPageOnyxProps = {
+    /** New group chat draft data */
+    newGroupDraft: OnyxEntry<OnyxTypes.NewGroupChatDraft>;
+
+    /** All of the personal details for everyone */
+    allPersonalDetails: OnyxEntry<OnyxTypes.PersonalDetailsList>;
+};
+
+type NewChatConfirmPageProps = NewChatConfirmPageOnyxProps;
+
+type Section = ListItem & {value: string};
```

Comment:
Checking this again, `ListItem` has `login` prop we can use that instead of `value`